ksqldb-functional-tests/src/test/resources/historical_plans/group-by_-_AVRO_group_by_struct/6.2.0_1607703224334/plan.json

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
Adding a note for posterity: all the historic plan changes here appear to be the removal of UNWRAP_SINGLES from stream select key steps, which happens when grouping by a different key or when streams are repartitioned for joins. This makes sense in light of the changes in this PR.

Reply:
Thanks for checking this! that was my next step actually to make sure it works 😂